storage/aptosdb/src/state_merkle_db.rs

```diff
@@ -293,7 +308,14 @@ impl StateMerkleDb {
                 .collect::<Vec<_>>()
                 .par_iter()
                 .with_min_len(128)
-                .map(|(node_key, node)| batch.put::<JellyfishMerkleNodeSchema>(node_key, node))
+                .map(|(node_key, node)| {
```

Comment:
what is the 128? might be worth of creating a const for it?

Reply:
Just a random number that is not too small. Didn't spend time on tuning this.